docker/php/www.conf

```diff
@@ -440,4 +440,19 @@ pm.max_spare_servers = 3
 
 php_admin_flag[log_errors] = on
 
-php_admin_value[date.timezone] = Europe/Paris
\ No newline at end of file
+php_admin_value[error_reporting] = E_ALL
+php_admin_value[display_errors] = on
+php_admin_value[display_startup_errors] = On
+php_admin_value[log_errors_max_len] = 1024
+
+php_admin_value[date.timezone] = Europe/Paris
+
+; Debug Settings
+php_admin_value[xdebug.mode] = develop
+php_admin_value[xdebug.start_with_request] = yes
+php_admin_value[xdebug.log]= 2>&1
```

Comment:
/dev/stdout ?

Reply:
I was wonder if it's usefull to put the log... I was just editing the commit on my phone to remove this line but you solution is better

Reply:
Done